data/json/recipes/recipe_ammo.json

```diff
@@ -227,7 +227,7 @@
       ["crucible_clay", -1]
      ],
      [
-      ["forge", 25], ["char_forge", 5]
+      ["forge", 25]
```

Comment:
Now the charcoal requires just as many charges the electric forge does. Previously it would just require 1/5 of the charges, it this still balanced? The same question has to be asked about all the recipes where the char_forge got removed and only the electric forge remains.